test/e2e/common/node/container_probe.go

```diff
@@ -821,6 +821,26 @@ func getRestartCount(p *v1.Pod) int {
 	return count
 }
 
+func testGRPCServerPodSpec(readinessProbe, livenessProbe *v1.Probe, containerName string, port int) *v1.Pod {
```

Comment:
what's the reason for reordering? It's hard to validate changes

Reply:
Sorry, I just forget to remove this